lightkurve/lightcurve.py

```diff
@@ -695,6 +695,16 @@ def normalize(self, unit='unscaled'):
 
         return lc
 
+    def trim(self, trim_mask):
+        """Trims cadences form a light curve using trim_mask
+
+        Returns
+        -------
+        trimmed_lightcurve : `LightCurve`
+            A new light curve object from where trimmed cadences have been removed.
+        """
+        return self[trim_mask]  # This will return a sliced copy
```

Comment:
Do we need a dedicated `trim()` method?  One can already do `lc = lc[trim_mask]` instead of `lc = lc.trim(trim_mask)`.  It's usually better to avoid having multiple ways to do the same thing.

Reply:
I didn't realize that when I wrote that code. I will eliminate the trim method.